### PR TITLE
move perspectivesBeta to main perspectives page

### PR DESCRIPTION
**Pull request by jgraff2**

### CODE REVIEW

File: view/perspective/app.js [hunk 1]
```diff
@@ -96,7 +99,7 @@ function handleError(err) {
   }
 
   ERROR_INFO_DIV.innerHTML = msg;
-  u.removeSpinner(SPINNER_ID);
+  removeSpinner();
```
Comment (iamigo) [inline view/perspective/app.js]: reverting? let's make sure we keep any recent changes made since you forked the page

Comment (jgraff2) [reply view/perspective/app.js]: oops...
fixed.

**Comment by pallavi2209** [comment]
@jgraff2 Can you please resolve the conflicts? We can merge this once that is resolved.

**Comment by jgraff2** [comment]
@pallavi2209 done.

**Pull request merged**